extension/src/testConfigManager.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fs from 'fs';
+import * as path from 'path';
+import { window, workspace, ExtensionContext, TextEditor } from 'vscode';
+
+import { ProjectInfo, ProjectManager } from './projectManager';
+import * as Configs from './Constants/configs';
+import { TestConfig } from './Models/testConfig';
+import * as Logger from './Utils/Logger/logger';
+
+export class TestConfigManager {
+    private readonly _configPath: string;
+    constructor(context: ExtensionContext, private _projectManager: ProjectManager) {

```

Comment:
Why not directly pass in configPath?

Reply:
I'd like it to be managed by TestConfigManager, right now it uses extension path, but it could be other places